.github/workflows/build.yml

```diff
@@ -0,0 +1,46 @@
+name: Build
```

Comment:
What was the motivation to separate Build and test?

Reply:
If you look at GitHub reporting, now you get `Build / ...` and `Test / ...` separately. We don't need to build it for testing, so "Build" is a limited set of actions that don't run during testing. For example the WASM build has no test cases just a special build action.